hijack/settings.py

```diff
@@ -15,7 +14,7 @@
     {
         'name': 'HIJACK_AUTHORIZE_STAFF',
         'default': False,
-        'legacy_name': 'ALLOW_STAFF_TO_HIJACKUSER',
+        'legacy_name': None,
```

Comment:
I don't understand this change well. Could you please explain?

Reply:
Good questions. I think this I just accidentally committed that, sorry. I should have spotted this myself.